packages/framework/presence/src/presence.ts

```diff
@@ -36,12 +36,24 @@ export type ClientSessionId = SessionId & { readonly ClientSessionId: "ClientSes
  * @alpha
  */
 export const SessionClientStatus = {
+	/**
+	 * The session client is connected to the delta stream.
+	 * Can send to and receive updates from other session clients.
+	 */
```

Comment:
Is the "delta stream" something we expect developers to know? Would it be as accurate to say, "connected to the Fluid service?" That's a term we use throughout our documentation, e.g. https://fluidframework.com/docs/concepts/architecture/#fluid-service

Reply:
Thanks I was struggling wording it, that sounds better